test/functional/notes_controller_test.rb

```diff
@@ -705,7 +705,7 @@ def teardown
     assert !(notes & questions).present?
```

Comment:
Just that using nodes(:question) is misleading because
the /notes page is designed not to return any question related notes

Reply:
on line 693 to 695 as well.